packages/terra-folder-tree/tests/jest/FolderTree.test.jsx

```diff
@@ -1,66 +1,85 @@
 import React from 'react';
 import ActionHeader from 'terra-action-header';
 /* eslint-disable-next-line import/no-extraneous-dependencies */
-import { shallowWithIntl } from 'terra-enzyme-intl';
+import { mountWithIntl, shallowWithIntl } from 'terra-enzyme-intl';
 import FolderTree from '../../src/FolderTree';
+import FolderTreeItem from '../../src/subcomponents/FolderTreeItem';
```

Comment:
Can we import these from index.js? This way, the usage would be more in line with how a consumer would utilize it.